src/lib/builtins/array.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{
```

Comment:
Are the changes in this file related to the implementation for `Symbols`, or are they just refactoring?

Reply:
Both really.
I needed to do some refactoring around the contruction of arrays